substrate/client/network/sync/src/lib.rs

```diff
@@ -2641,104 +2641,104 @@ mod test {
 	/// have) from it. In the meanwhile peer 2 sends us block 4 and 3 and we send another request
 	/// for block 2 to peer 2. Peer 1 answers with block 2 and then peer 2. This will need to
 	/// succeed, as we have requested block 2 from both peers.
-	#[test]
-	fn do_not_report_peer_on_block_response_for_block_request() {
```

Comment:
I don't think this test makes sense if the PR is merged, because the logic is changed so we don't request block 2 from peer 2 anymore.